src/components/HomepageFeatures/styles.module.css

```diff
@@ -43,10 +47,11 @@
   box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
   margin-bottom: 20px;
   transition: transform 0.2s ease-in-out;
-  height: 300px;
   display: flex;
   flex-direction: column;
   justify-content: space-between;
+  color: #000;
```

Comment:
When color values map to specific color, the color name should be used.
`color: black;`

Reply:
Thanks! Changes made here: https://github.com/NMFS-RADFish/radfish/pull/106/commits/dc9b9cc887c7dbb2c5be6fa2aa397ac8d20aa7a7